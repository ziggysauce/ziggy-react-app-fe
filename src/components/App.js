import React from 'react';
import Credit from './Credit';

const App = () => (
  <div className="app-container">
    <h1>React App Boilerplate</h1>
    <Credit />
  </div>
);

export default App;
