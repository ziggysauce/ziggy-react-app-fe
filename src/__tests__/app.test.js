import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/App';

describe('Component testing', () => {
  test('App component renders with h1 text and Credit component', () => {
    const component = renderer.create(<App></App>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

