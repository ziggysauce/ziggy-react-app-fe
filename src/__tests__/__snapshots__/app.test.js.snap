// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component testing App component renders with h1 text and Credit component 1`] = `
<div
  className="app-container"
>
  <h1>
    React App Boilerplate
  </h1>
  <div
    style={
      Object {
        "boxShadow": "0px 0px 10px 2px lightgrey",
        "display": "flex",
        "flexDirection": "column",
        "fontFamily": "sans-serif",
        "padding": "30px",
      }
    }
  >
    <div>
      <h2>
        Instructions to get started:
      </h2>
      <p>
        Delete 
        <code
          style={
            Object {
              "backgroundColor": "lightgrey",
              "padding": "3px",
            }
          }
        >
          Credit.js
        </code>
         file
      </p>
      <p>
        Edit 
        <code
          style={
            Object {
              "backgroundColor": "lightgrey",
              "padding": "3px",
            }
          }
        >
          App.js
        </code>
         file
      </p>
      <p>
        Edit 
        <code
          style={
            Object {
              "backgroundColor": "lightgrey",
              "padding": "3px",
            }
          }
        >
          _app.scss
        </code>
         file
      </p>
    </div>
    <div>
      <h3>
        Follows and stars are much appreciated!
      </h3>
      <iframe
        frameBorder="0"
        height="30px"
        scrolling="0"
        src="https://ghbtns.com/github-btn.html?user=ziggysauce&type=follow&count=true&size=large"
        width="220px"
      />
      <iframe
        frameBorder="0"
        height="30px"
        scrolling="0"
        src="https://ghbtns.com/github-btn.html?user=ziggysauce&repo=ziggy-react-app&type=star&count=true&size=large"
        width="160px"
      />
    </div>
  </div>
</div>
`;
